dev-lang/fpc/fpc_bin-3.0.0.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Pre-built binaries for the FPC build"
+DESCRIPTION="Pre-built FPC binaries for Haiku i386."
+HOMEPAGE="http://www.freepascal.org"
+SOURCE_URI="http://sourceforge.net/projects/freepascal/files/Haiku/3.0.0/fpc-3.0.0.i386-haiku.tar"
+CHECKSUM_SHA256="a81bf8ca23472bf04a7934c3c881c5c26d1a3a801c36360a9a667d1625d8d1f2"
+LICENSE="
+	GNU LGPL v3
+	GNU GPL v3
+	"
+COPYRIGHT="1993-2013 by Florian Klaempfl"
```

Comment:
Remove the 'by'. Also these are all out of order; see https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines for proper ordering